modules/checking-mco-status.adoc

```diff
@@ -14,35 +14,53 @@ To see the status of the Machine Config Operator, its sub-components, and the re
 [source,terminal]
 ----
 $ oc get machineconfigpool
-NAME      CONFIG                  UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
-master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
-worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
 ----
 +
-In the previous output, there are three master and three worker nodes. All machines are updated and none are currently updating. Because all nodes are `Updated` and `Ready` and none are `Degraded`, you can tell that there are no issues.
-
-. To see each existing `machineconfig`, type:
-+
 [source,terminal]
```

Comment:
You could add an ".Example output" line before this one. For more information, see [Code blocks, command syntax, and example output](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).

Reply:
@pneedle-rh Thanks! I split a few command/output code blocks, but forgot to add the .Example output!